SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -1712,6 +1818,18 @@ class CVariable {
    */
   inline virtual void SetmuT(unsigned long iPoint, su2double val_muT) {}
 
+  /*!
+   * \brief Get the value of the desired grid size for Scale Resolving Simulations.
+    * \return the value of the desired grid size for Scale Resolving Simulations.
+  */
+  inline virtual su2double GetSRSGridSize(unsigned long iPoint) const { return 0.0; }
+
+  /*!
+   * \brief Set the value of the desired grid size for Scale Resolving Simulations.
+   * \param[in] val_gridSize
+   */
+  inline virtual void SetSRSGridSize(unsigned long iPoint, su2double val_gridSize) {}
```

Comment:
If this is one of the debug quantities you are removing please move these accessors to the turbulence variable (thus not virtual anymore).